scripts/calibrate_vcs.py

```diff
@@ -653,16 +653,20 @@ def write_cont_scripts(self):
                         "nodes": "{0}".format(nnodes),
                         "gres": "gpu:1",
                         "ntasks-per-node": "1"}
-        module_list = ["RTS/master"]
+        module_list = ["RTS/chj"]
         commands = list(self.script_body)  # make a copy of body to then extend
+        #commands.append("module use /pawsey/mwa/software/python3/modulefiles")
+        #commands.append("module load RTS/master")
         commands.append("srun --export=all -N {0} -n {0} rts_gpu {1}".format(nnodes, fname))
         jobid = submit_slurm(rts_batch, commands,
                                 slurm_kwargs=slurm_kwargs,
                                 module_list=module_list,
                                 batch_dir=self.batch_dir,
                                 submit=self.submit,
                                 queue='gpuq',
-                                export="NONE")
+                                export="NONE",
+                                module_dir='/pawsey/mwa/software/python3/modulefiles',
```

Comment:
I don't think so because it's not a file that it is loading. I do have a module dir in config.py but this is different because '/pawsey/mwa/software/python3/modulefiles' is where the garrawarla RTS version is installed

Reply:
I'm just thinking that if we have a single location to store the module directory it'll be easier to change in the future

Reply:
So to be clear, we do have a module dir defined in config.py
`'module_dir' : '/group/mwa/software/modulefiles',`
So this is the single location to store the module dir. The above dir is where all the galaxy software is stored. For the RTS to run on Garrawarla I need to use a different module dir, hence why I manually input it here. 

Perhaps I could make some option somewhere to tell it if you want to send a job to Galaxy of Garrawarla but that might make a mess of things.

Reply:
Yeah that's a little more messy than i thought it would be. Probably just leave it for now and we'll update it if we move off of galaxy entirely

Reply:
Agreed